.gitignore

```diff
@@ -1,2 +1,12 @@
 **/.idea
 .env
+
+*~
+\#*\#
+/.emacs.desktop
```

Comment:
is it better to add these to your local git config?

Reply:
Probably, but it seemed like there were existing editor-specific directories in here so I followed convention. :D 

Will change this.